src/app/view/web/WebChessView.ts

```diff
@@ -45,6 +45,26 @@ export class WebChessView implements ChessBoardView {
     });
   }
 
+  movePiece(squareFrom: string, squareTo: string): void {
+    const divFrom = this.parent.querySelector(`#${squareFrom}`);
+    const divTo = this.parent.querySelector(`#${squareTo}`);
+    const pieceImage = divFrom?.firstChild;
+
+    if (pieceImage) {
+      divFrom?.removeChild(pieceImage);
+      divTo?.appendChild(pieceImage);
```

Comment:
Tu musi być ten znak zapytania? jest szansa na to, ze jeśli pieceImage będzie undefined to to z bloku się wykona?